os/drivers/wireless/virtual/vdev.c

```diff
@@ -303,12 +303,6 @@ void vwifi_start(void)
 	return;
 }
 
-void up_netinitialize(void)
```

Comment:
Is it okay to remove this function?
As I know, this function is required for every board/driver which uses from `up_initialize()`.

Reply:
the definition is already present at amebad_enet.c, so it is giving conflicts

Reply:
@pillip8282 Could you review this deletion?

Reply:
if you disable virtual driver option then this file isn't built. You can't remove this function

Reply:
As @pillip8282  said, we should not remove this. Please revert it.